### ISSUE TITLE
I cant finish a quest with mine blocks

### ISSUE THREAD

**Issue by meepoo397** [ToroCraft/ToroQuest#133]
i mine it with his pickaxe 15 times but it said "you are 15 short"

**Comment by frodare** [comment]
Make sure you are below level 30.  You will know it is working when the item stack says something like "Iron Ore for Civilization" when you hover your mouse over it.

**Comment by NekoAmps** [comment]
I'm having the same issue as OP. When a quest requires me to collect resources with a loaner tool, the blocks that I collect with the tool are normal blocks, which are not recognized in the lefthand column of the Village Lord's quest GUI. So I can't complete those quests, but I am able to complete collection quests that don't require a loaner tool. This occurs in any world, with and without additional mods, and I was well under level 30 during each occurrence.

Affected Torocraft versions: all versions for MC 1.11.2, haven't tested others
Forge version: 1.11.2-13.20.0.2315
Java:  v8 u131
OS: Windows 10, 64-bit

**Comment by frodare** [comment]
Thanks posting this issue here.  I will reopen it and try figure out what is going on with it.

**Comment by Wing01** [comment]
We have been unable to recreate this issue thus far. If someone comes across it again, we will take another look.

**Comment by dimion123** [comment]
I've had this same issue, too.

**Comment by Johannason** [comment]
Also experiencing this issue. He wants 23 Redstone Ore blocks and has given me a diamond, Silk Touch pickaxe to retrieve them with. I collected almost a full stack of redstone ore with the pick, and none of them show as "ore for civilization". They're just normal ore blocks.

**Comment by frodare** [comment]
@Johannason did you mine the blocks deep under ground, below y=30?

**Comment by dimion123** [comment]
@frodare Don't know about others, but I did.

**Comment by frodare** [comment]
Ok I am going to reopen this one, maybe we will figure out what is going on one day, lol!

**Comment by Johannason** [comment]
Yes, I dug below 30. He wanted Redstone Ore, which doesn't appear until 16.
It may be worth noting that this happened in multiplayer, and both I and the server host had the same quest and the same issue.
In singleplayer, I was able to deliver Iron Ore and Stone just fine.

**Comment by frodare** [comment]
@Johannason maybe that is the issue!  I will test in server mode when I get a chance to.

**Comment by Mijithemidge** [comment]
This is a problem that I am having as well.  He gave me a pick and told me to get coal ore, but he didn't accept it. Same happened when he gave me a shovel and told me to get gravel.

**Comment by Gaiagirl1** [comment]
I’ve been trying to figure out how to complete the mine quest but when I finish it the game tells me I need X amount of blocks that I already have in my inventory

**Comment by ProsperCraft** [comment]
I just completed the coal quest.  It works.

**Comment by sideshowbo12** [comment]
single player world redstone quest does not accept the ore.

**Comment by bluebear94** [comment]
I also ran into the issue. In addition, I did turn in the loaner pickaxe, but when I try to abandon the quest, the lord still asks me for it or 5 emeralds.

**Comment by Jaluco** [comment]
same issue, doesnt matter what i do it only recognizes one block

**Comment by Zokrace** [comment]
Had the issue that i accidentally turned in the loaner pickaxe . neither able to complete or abandon quest as in both cases lord wants his named pickaxe back
any way to reset the quest or recreate the named pickaxe?

**Comment by Tomato-No-Mae** [comment]
i've just had the same issue happen to me, in a single player world, I was definitely below y=30 and I was wondering if there was a command to force the quest to complete

it may have something to do with the variant of iron ore, because this is in a modpack with ore variants for different stone types

**Comment by James-MacPhee** [comment]
Still having the same issue. It's quite annoying as I gave him the loaner tool and now he won't accept my redstone. Would be nice to have this fixed after 3 YEARS of this issue being open

**Comment by 69raccoon96** [comment]
Same issue as James-MacPhee having, can't give redstone
![image](https://user-images.githubusercontent.com/50990891/106053716-ee460480-610c-11eb-8b86-283c05de4a5c.png)
